buildchain/buildchain/salt_tree.py

```diff
@@ -291,6 +291,8 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/addons/nginx-ingress-control-plane/deployed/chart.sls'),
     Path('salt/metalk8s/addons/nginx-ingress-control-plane/deployed/',
          'tls-secret.sls'),
+    Path('salt/metalk8s/addons/nginx-ingress-control-plane/',
+         'control-plane-ip.jinja'),
```

Comment:
Two things:

- this commit could introduce the helper with the code that uses it, but not critical
- I don't see why it should use the `.jinja` extension, instead of `.sls` - it relies on `pillar` and `grains` (that one could setup manually, sure), so users can't expect it to work with "simple" Jinja templating